components/gitpod-db/go/encryption.go

```diff
@@ -14,11 +14,19 @@ import (
 	"fmt"
 )
 
-type Cipher interface {
+type Encryptor interface {
 	Encrypt(data []byte) (EncryptedData, error)
+}
+
+type Decryptor interface {
 	Decrypt(data EncryptedData) ([]byte, error)
 }
 
+type Cipher interface {
+	Encryptor
+	Decryptor
+}
```

Comment:
To have more fined-grained access to the relevant parts. We don't always need the full Cipher interface, and often only need one of the the two parts.